# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sjinn/disable-profiling.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sjinn/Makefile.vanilla	2010-07-01 09:43:48.000000000 +0200
+++ sjinn/Makefile	2010-07-01 09:44:02.000000000 +0200
@@ -1,5 +1,6 @@
 CC=gcc
-CFLAGS=-pg -Wall
+CFLAGS=-Wall
+#CFLAGS=-pg -Wall
 #CFLAGS=-g
 LIB=-lm
 INC=
